website/docs/index.html.markdown

```diff
@@ -116,6 +131,7 @@ The following arguments are supported:
 * `password` - (Optional) Password for the given user, if that user has a password, can also be sourced from the `MYSQL_PASSWORD` environment variable.
 * `proxy` - (Optional) Proxy socks url, can also be sourced from `ALL_PROXY` or `all_proxy` environment variables.
 * `tls` - (Optional) The TLS configuration. One of `false`, `true`, or `skip-verify`. Defaults to `false`. Can also be sourced from the `MYSQL_TLS_CONFIG` environment variable.
+* `custom_tls` - (Optional) Sets custom tls options for the connection. This is a block containing an optional `config_key` and the following required arguments: `ca_cert`, `client_cert` and `client_key`. Documentation for encrypted connections can be found [here](https://dev.mysql.com/doc/refman/8.0/en/using-encrypted-connections.html).
```

Comment:
Consider advising people to set lower `connect_retry_timeout_sec` for debugging. I tested this and it takes "infinity" (default is 10 minutes) to fail in case there is some issue.
Maybe specify that `ca_cert` and so on as further bullets nested also this.

Also, does `config_key` matter at all? Maybe it should be explicitly said we don't care (it's not specified anywhere in MySQL docs).